src/xamarin-client/Adaptive.ReactiveTrader.Client.Domain/Transport/Wamp/ServiceObservableExtensions.cs

```diff
@@ -93,7 +93,7 @@ public static class ServiceObservableExtensions
                 disposables.Add(
                     source.Subscribe(innerSource =>
                     {
-                        IObservable<TValue> innerSourcePublished = innerSource.Replay(1).RefCount();
+                        IObservable<TValue> innerSourcePublished = innerSource.Publish().RefCount();
```

Comment:
Honestly, I don't know, it was coming from the last Xamarin changes in James' branch. Would it make a difference?